src/tabs/tab-header-group.component.ts

```diff
@@ -86,6 +87,19 @@ export class TabHeaderGroup implements AfterContentInit, OnDestroy {
 	@Input() contentAfter: TemplateRef<any>;
 
 	@Input() contentBefore: TemplateRef<any>;
+
+		/**
+	 * Set to 'true' to have all pane references associated with each tab header
+	 * in the tab header group cached and not reloaded on tab switching.
+	 */
+	@Input() set cacheActive(shouldCache: boolean) {
+		this._cacheActive = shouldCache;
+	}
+
+	get cacheActive() {
+		return this._cacheActive;
+	}
```

Comment:
If this has no other effect, could it just be a normal property? i.e. not a getter/setter?

Reply:
True, since the input is being changed directly and that would get picked up by angular's change detection.